test/Conversion/MooreToCore/basic.mlir

```diff
@@ -425,134 +425,6 @@ moore.module @Struct(in %a : !moore.i32, in %b : !moore.i32, in %arg0 : !moore.s
-    // CHECK:   cf.br ^[[BB3:.+]]
-    // CHECK: ^[[BB3]]:
-    // CHECK:   [[B_PRB:%.+]] = llhd.prb [[B]]
-    // CHECK:   [[C_PRB:%.+]] = llhd.prb [[C]]
-    // CHECK:   [[RES:%.+]] = comb.add [[B_PRB]], [[C_PRB]] : i1
-    // CHECK:   [[T0:%.+]] = llhd.constant_time <0ns, 1d, 0e>
-    // CHECK:   llhd.drv [[D]], [[RES]] after [[T0]]
-    // CHECK:   cf.br ^[[BB1]]
-    %br = moore.read %b : <i1>
-    %cr = moore.read %c : <i1>
-    %0 = moore.add %br, %cr : i1
-    moore.nonblocking_assign %d, %0 : i1
-    moore.return
-  }
-
-  // CHECK: llhd.process
-  moore.procedure always {
-    // CHECK:   cf.br ^[[BB1:.+]]
-    // CHECK: ^[[BB1]]:
-    // CHECK:   [[C_OLD:%.+]] = llhd.prb [[C]]
-    // CHECK:   llhd.wait ([[C]] : !hw.inout<i1>), ^[[BB2:.+]]
-    // CHECK: ^[[BB2]]:
-    // CHECK:   [[C_CURR:%.+]] = llhd.prb [[C]]
-    // CHECK:   [[NOT_C_OLD:%.+]] = comb.xor [[C_OLD]], %true
-    // CHECK:   [[POSEDGE:%.+]] = comb.and [[NOT_C_OLD]], [[C_CURR]] : i1
-    // CHECK:   [[C_CURR1:%.+]] = llhd.prb [[C]]
-    // CHECK:   [[NOT_C_CURR:%.+]] = comb.xor [[C_CURR1]], %true
-    // CHECK:   [[NEGEDGE:%.+]] = comb.and [[C_OLD]], [[NOT_C_CURR]] : i1
-    // CHECK:   [[PROCEED:%.+]] = comb.or [[POSEDGE]], [[NEGEDGE]] : i1
-    // CHECK:   cf.cond_br [[PROCEED]], ^[[BB3:.+]], ^[[BB1]]
-    // CHECK: ^[[BB3]]:
-    // CHECK:   [[RES:%.+]] = comb.add [[B_PRB:%.+]], [[C_PRB:%.+]] : i1
-    // CHECK:   [[T0:%.+]] = llhd.constant_time <0ns, 1d, 0e>
-    // CHECK:   llhd.drv [[D]], [[RES]] after [[T0]]
-    // CHECK:   cf.br ^[[BB1]]
-    moore.wait_event edge %cr : i1
-    %0 = moore.add %br, %cr : i1
-    moore.nonblocking_assign %d, %0 : i1
-    moore.return
-  }
-
-  // CHECK: [[B_PRB]] = llhd.prb [[B]]
-  // CHECK: [[C_PRB]] = llhd.prb [[C]]
-  %br = moore.read %b : <i1>
-  %cr = moore.read %c : <i1>
-  // CHECK: llhd.process
-  moore.procedure always {
-    // CHECK:   cf.br ^[[BB1:.+]]
-    // CHECK: ^[[BB1]]:
-    // CHECK:   [[C_OLD:%.+]] = llhd.prb [[C]]
-    // CHECK:   llhd.wait ([[C]] : !hw.inout<i1>), ^[[BB2:.+]]
-    // CHECK: ^[[BB2]]:
-    // CHECK:   [[C_CURR:%.+]] = llhd.prb [[C]]
-    // CHECK:   [[NOT_C_CURR:%.+]] = comb.xor [[C_CURR]], %true
-    // CHECK:   [[NEGEDGE:%.+]] = comb.and [[C_OLD]], [[NOT_C_CURR]] : i1
-    // CHECK:   [[PROCEED:%.+]] = comb.or [[NEGEDGE]] : i1
-    // CHECK:   cf.cond_br [[PROCEED]], ^[[BB3:.+]], ^[[BB1]]
-    // CHECK: ^[[BB3]]:
-    // CHECK:   [[RES:%.+]] = comb.add [[B_PRB]], [[C_PRB]] : i1
-    // CHECK:   cf.cond_br %cond, ^[[BB4:.+]], ^[[BB5:.+]]
-    // CHECK: ^[[BB4]]:
-    // CHECK:   [[T0:%.+]] = llhd.constant_time <0ns, 1d, 0e>
-    // CHECK:   llhd.drv [[D]], [[RES]] after [[T0]]
-    // CHECK:   cf.br ^[[BB1]]
-    // CHECK: ^[[BB5]]:
-    // CHECK:   cf.br ^[[BB1]]
-    moore.wait_event negedge %cr : i1
-    %0 = moore.add %br, %cr : i1
-    cf.cond_br %cond, ^bb1, ^bb2
-  ^bb1:
-    moore.nonblocking_assign %d, %0 : i1
-    moore.return
-  ^bb2:
-    moore.return
-  }
-}
```

Comment:
This removes tests for the assign operations without adding replacements.

Reply:
Ugh thanks for catching that! 👍